torchgeo/datamodules/quakeset.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""QuakeSet datamodule."""
+
+from typing import Any
+
+import kornia.augmentation as K
+import torch
+
+from ..datasets import QuakeSet
+from ..transforms import AugmentationSequential
+from .geo import NonGeoDataModule
+
+
+class QuakeSetDataModule(NonGeoDataModule):
+    """LightningDataModule implementation for the QuakeSet dataset.
+
+    .. versionadded:: 0.6
+    """
+
+    mean = torch.tensor(0.0)
```

Comment:
would it help if we actually compute some reasonable normalization statistics for the challenge based on the train set? or should we leave that to participants to decide how they scale data?

Reply:
We can do this but I wasn't sure if that's the best way to normalize Sentinel-1 bands